Cargo.toml

```diff
@@ -68,12 +68,14 @@ ansi_term = "0.12"
 atty = "0.2"
 buildstructor = "0.3"
 calm_io = "0.1"
+camino = "1"
```

Comment:
camino stuff is re-exported in `saucer` so you can import anything you need from there